src/core/MRSystem.js

```diff
@@ -111,6 +97,17 @@ export class MRSystem {
      */
     eventUpdate() {}
 
+    /**
+     *
+     * @param entity
+     */
```

Comment:
look at the jsdoc warnings in the files section of github - make sure theyre filled out